sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample06_RequestingStorageServiceVersions.md

```diff
@@ -4,6 +4,10 @@ This sample demonstrates configuring the Blob Storage client to use a specific v
 
 To begin, please ensure that you're familiar with the items discussed in the [Event Processor Handlers](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample03_EventProcessorHandlers.md) sample.  You'll also need to have the prerequisites and connection string information available, as discussed in the [Getting started](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples#getting-started) section of the README.
 
+## Table of contents
```

Comment:
I was torn on this one but ultimately landed on wanting a consistent format. As you stated, it will set the expectation that the table needs to be updated as new content is added. This could be a possible oversight later and is easily prevented by including the table now.